src/Sulu/Bundle/MediaBundle/DependencyInjection/ImageFormatCompilerPass.php

```diff
@@ -74,13 +78,18 @@ protected function setFormatsFromFile($fullPath, &$activeFormats, $defaultOption
         $fileName = basename($fullPath);
 
         $locator = new FileLocator($folder);
-        $loader = new XmlFormatLoader($locator);
-        $loader->setDefaultOptions($defaultOptions);
+
+        $xmlLoader10 = new XmlFormatLoader10($locator);
+        $xmlLoader11 = new XmlFormatLoader11($locator);
+        $xmlLoader10->setGlobalOptions($defaultOptions);
+        $xmlLoader11->setGlobalOptions($defaultOptions);
+
+        $resolver = new LoaderResolver([$xmlLoader10, $xmlLoader11]);
+        $loader = new DelegatingLoader($resolver);
```

Comment:
This is not that easy, as the `XmlFormatLoader10` and `XmlFormatLoader11` need a locator which is initialized with the folder. We would have to pass an array of all folders to the locator.